### PR TITLE
fixed sytax typos, added express install to darwin deploy.

### PR DESCRIPTION
**Pull request by ChrisForti**

### CODE REVIEW

File: darwin-deploy.sh [hunk 1]
```diff
@@ -25,12 +25,12 @@ fi
...
-if (which node)
+if (which express)
 then 
- echo "node is installed"
+ echo "express is installed"
 else
-  echo "installing node"
-brew install node
+  echo "installing express"
+brew install express
```
Comment (stephenlauck) [inline darwin-deploy.sh]: Express is actually a javascript library. So you should follow this guide on how to install express as a javascript npm dependency in a javascript project:

https://expressjs.com/en/starter/installing.html

**Review by stephenlauck — CHANGES_REQUESTED** [review]

File: darwin-deploy.sh (hunk 1 above)
Comment (ChrisForti) [reply darwin-deploy.sh]: I know this, I just forgot to change the pkg manager to npm. I even tested it with npm, and then copied the node if statement, and didn't change the most important part. Thank you sir

File: darwin-deploy.sh [hunk 2]
```diff
@@ -25,12 +25,12 @@ fi
 
 # Express should be installed
 
-if (which node)
+if (which express)
```
Comment (stephenlauck) [inline darwin-deploy.sh:28]: You will need to use npm to check if express is installed using something like:

```
npm ls | grep express
```

Express is not a system package that you're installing in your system (like apt package) but is internal to your javascript and managed by npm.

**Review by stephenlauck — CHANGES_REQUESTED** [review]

File: darwin-deploy.sh (hunk 2 above)
Comment (ChrisForti) [reply darwin-deploy.sh:28]: Oh i see, i didnt actually check that. Which i just did `├── express@4.18.2`. So should i change that? From what your saying, in your last comment. It sounds like i just need axcess to the the modules express provides, and does not belong in the deploy script? We have already set it as a dependency in the first couple of code in this repo.

Comment (stephenlauck) [reply darwin-deploy.sh:28]: Let's find some time to do a vid chat review of how to manage dependencies for node deploys.

Comment (ChrisForti) [reply darwin-deploy.sh:28]: I forgot about this, is this something we still need to do?

Comment (ChrisForti) [reply darwin-deploy.sh:28]: I am ready when you are my friend.

**Pull request closed without merging**